common/components/Input.js

```diff
@@ -69,6 +71,13 @@ const Input = ({
         inputIsValid = false;
         setErrorMessage(`최대 값은 ${max}${unit} 입니다`);
       }
+
+      if (value.indexOf('.') > dotIndex) {
```

Comment:
```suggestion
      if (value.indexOf('.') !== -1 ) {
```
개인적인 생각일지 모르나  해당 코드를 보는 사람이 indexOf 의 메서드를 이해하는 가정 선에서는 굳이 dotIndex 라는 상수로 해당 숫자를 처리 해주지 않아도 좋을 것 같습니다. indexOf의 기본 사양은 해당 사항이 없으면 -1를 반환하는데 dotIndex가 -1로 할당이 되어 있으면 오히려 소수점 부분의 위치 파악에 혼란을 겪을거 같습니다.

Reply:
그렇군요 상수 관련에 대해서는 const로 할당 해야 된다고 생각했었는데 이 부분은 혜민님께도 의견 들어보고 수정해보도록 하겠습니다